client/my-sites/plans/current-plan/ecommerce-trial/index.tsx

```diff
@@ -191,10 +187,7 @@ const ECommerceTrialCurrentPlan = () => {
 							  ) }
 					</p>
 					<Button className="e-commerce-trial-current-plan__trial-card-cta" primary>
-						{
-							// TODO: translate when final copy is available
-							'Get Commerce'
-						}
+						{ translate( 'Upgrade now' ) }
 					</Button>
```

Comment:
Can we add a click handler for this CTA? It should be enough to redirect the user to `/checkout/ecommerce-bundle-monthly`. It would also be good to instrument it with a Tracks event.